Blocks/src/main/java/com/google/blocks/AbstractProgrammingModeActivity.java

```diff
@@ -33,11 +33,10 @@
  * @author lizlooney@google.com (Liz Looney)
  */
 public abstract class AbstractProgrammingModeActivity extends ThemedActivity {
+    private final Timer timer = new Timer();
```

Comment:
Spacing issue

Reply:
Also why does this need to be moved?

Reply:
I accidentally ran my formatter on changed files, I'll undo the spacing and order changes.